src/EnergyPlus/StandardRatings.cc

```diff
@@ -1604,10 +1611,12 @@ namespace StandardRatings {
             // Calculate SEER value from the Energy Efficiency Ratio (EER) at the AHRI test conditions and the part load factor.
             // First evaluate the Part Load Factor curve at PLR = 0.5 (AHRI Standard 210/240)
             PartLoadFactor = CurveValue(PLFFPLRCurveIndex, PLRforSEER);
+            PartLoadFractionDefault = 1.0 - CyclicDegradationCoeff * (1.0 - PLRforSEER);
```

Comment:
for single speed DX cooling coil added PLF calculation using the AHRI default PLF curve and C_D value of 0.25